__tests__/app.test.js

```diff
@@ -0,0 +1,28 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/development-data/index");
+
+beforeEach(() => {
+    return seed(data);
+  });
+  
+  afterAll(() => {
+    return db.end();
+  });
+
+  describe("GET /api/topics", () => {
+    test("Check topic for slug and description properties", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+            console.log(body, "<<< body");
```

Comment:
when checking objects in the topics array using a for each we need to check the length of the topics array beforehand.  An empty array will mean a forEach will not run, and this test will return a false pass.